### PR TITLE
Updated questions for faq page

### PR DESCRIPTION
**Pull request by chkumar246**
* Added communication channel
* Fixed Conference dates

### CODE REVIEW

**Comment by DollarAkshay** [comment]
Tested it out locally. Looks good to me

File: faq.html [hunk 1]
```diff
@@ -141,35 +150,37 @@ <h5><b>Q. How do I upload presentations to the site?</b></h5>
...
               <li>- 5.30 PM.</li>
           </ul>
         </div> 
         <div class="pycon-faq">
-          <h5><b>Q. What all does the Conference ticket (September 24 & 25) include? </b></h5>
+          <h5><b>Q. What all does the Conference ticket (Aug 22 to Aug 24, 2018) include? </b></h5>
           <ul>
-                <li>A. - Full conference pass for two days.</li>
+                <li>A. - Full conference pass for three days.</li>
                 <li>- Breakfast, Lunch and Hi-Tea.</li>
```
Comment (vnbang2003) [inline faq.html]: We haven't yet decided on breafast so let not add for now

File: faq.html [hunk 2]
```diff
@@ -141,35 +150,37 @@ <h5><b>Q. How do I upload presentations to the site?</b></h5>
...
-                <li>- PyCon India T-shirt which will be distributed on Sep 25.</li>
+                <li>- PyCon India T-shirt which will be distributed on Aug 25, 2018.</li>
```
Comment (SaptakS) [inline faq.html]: Is specifying the date necessary?

Comment (vnbang2003) [reply faq.html]: Yep people ask it

**Pull request merged**